src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -6778,6 +6778,15 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_ScopedRefAndRefStructOnly" xml:space="preserve">
     <value>The 'scoped' modifier can be used for refs and ref struct values only.</value>
   </data>
+  <data name="ERR_ScopedMismatchInParameterOfOverrideOrImplementation" xml:space="preserve">
+    <value>The 'scoped' declaration of parameter '{0}' doesn't match overridden or implemented member.</value>
+  </data>
+  <data name="ERR_ScopedMismatchInParameterOfTargetDelegate" xml:space="preserve">
```

Comment:
>ERR_ScopedMismatchInParameterOfTargetDelegate [](http://example.com/codeflow?start=14&length=45)

Do we need any checks for function pointer conversions?

Reply:
I think we won't need to because scoped can't be used in a function pointer parameter.

Reply:
There could still be a mismatch if the assigned method has a `scoped` parameter, although the resulting function pointer will be safe presumably because the method has a stronger contract than the function pointer.
```csharp
static R F1(R x, scoped R y) => x;
delegate*<R, R, R> d1 = &F1;
```